tempel.el

```diff
@@ -369,15 +402,15 @@ PROMPT is the optional prompt/default value."
           (push (cons mode (nreverse list)) result)))
       result)))
 
-(defun tempel--file-templates ()
-  "Return templates defined in `tempel-file'."
+(defun tempel--path-templates ()
+  "Return templates defined in `tempel-path'."
   (let ((mod (time-convert (file-attribute-modification-time
                             (file-attributes tempel-file))
```

Comment:
Here you still compute the modification time of `tempel-file`, while you should actually compute the modification time of all the files in `tempel-path`.